x-pack/plugins/security_solution/cypress/e2e/detection_alerts/changing_alert_status.cy.ts

```diff
@@ -80,6 +78,7 @@ describe('Changing alert status', () => {
                 'have.text',
                 `Selected ${numberOfAlertsToBeSelected} alerts`
               );
+              cy.get(TAKE_ACTION_POPOVER_BTN).should('exist');
```

Comment:
Did you add this because the test is flaky?

Reply:
There's a line on 84 as below, I think it wanted to check TAKE_ACTION_POPOVER_BTN is available here.
```
// cy.get(TAKE_ACTION_POPOVER_BTN).should('not.have.attr', 'disabled');
```